AppKit/CPCompatibility.j

```diff
@@ -179,7 +182,10 @@ else if (USER_AGENT.indexOf("AppleWebKit/") != -1)
         PLATFORM_BUGS |= CPJavaScriptPasteRequiresEditableTarget;
         // https://bugs.webkit.org/show_bug.cgi?id=39689
         PLATFORM_BUGS |= CPJavaScriptPasteCantRefocus;
+        PLATFORM_FEATURES[CPTextSizingAlwaysNeedsSetFontFeature] = YES;
     }
+	else
+        PLATFORM_ENGINE = CPBlinkBrowserEngine;
```

Comment:
@cacaodev i made Blink detection functional: http://stackoverflow.com/questions/20655470/how-to-detect-blink-in-chrome